ClientApp/src/components/Artefact/Artefact.js

```diff
@@ -0,0 +1,265 @@
+import React from 'react'
+
+import Divider from '@material-ui/core/Divider'
```

Comment:
Firstly, this is a backend issue that you are talking about and I asked you about certain parts regarding that display that was difficult for me to understand, but you dismissed it as a waste of time like what Jonah loves to say.
Secondly, most of the things that you discussed in Slack during that time were about the backend. This is a team project, since I am working for the frontend I think people who are working on the backend should stop making sudden changes without informing me regarding the design and I deserve to have a say in the design.
Thirdly, I disagree with you regarding the aesthetics. It should not be made a big deal at this point. If you have issues with the aesthetics, you should have complained in person when I show the demo and not creating an issue out of the blue.
With respect to material-ui, for the many times I have mentioned this, please do not make it an issue since I am just using some icons out of it. It can be easily substituted with another icon. However the ones from font awesome are not as suitable as the ones from material ui in my opinion.

Reply:
Please refrain from posting comments such as this, which are more to do with team dynamics than the code itself, in the public Github.

> It can easily be substituted with another icon
So *do* that. We have agreed to remove *all* of material UI from this project, no?

Unless the "later commit" you refer to extensively in successive comments is a different commit than  `0b8abf4 Updated code to align with code standard.`, most of the issues on this PR remain unresolved.